code/modules/clothing/masks/_masks.dm

```diff
@@ -82,3 +82,6 @@
 	if(user)
 		user.wear_mask_update(src, toggle_off = mask_adjusted)
 		user.update_action_buttons_icon() //when mask is adjusted out, we update all buttons icon so the user's potential internal tank correctly shows as off.
+
+/obj/item/clothing/mask/proc/consume_filter()
```

Comment:
This line does not meet our contribution standards, specifically for [dmdoc compliance](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md), and should be refactored.